src/index.scss

```diff
@@ -290,7 +289,18 @@
   --font-secondary: 'Inter', 'Roboto', sans-serif;
 }
 
+.dark-theme {
+  @include dark-mode();
+}
+
+@media (prefers-color-scheme: dark) {
+  .auto-theme {
+    @include dark-mode();
```

Comment:
this can be replaced with @ extend .dark-theme; if above suggestion is applied.

Not sure how this will work with media query.